indigo/distribution.yaml

```diff
@@ -1865,6 +1865,15 @@ repositories:
       type: git
       url: https://github.com/ZhuangYanDLUT/dlut_vision.git
       version: indigo-devel
+  drc_hubo:
+    doc:
+      type: git
+      url: https://github.com/JeongsooLim/drc_hubo.git
+      version: indigo-devel
```

Comment:
Your repository doesn't have a branch with that name. I assume you mean `master `?

Same below.